CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Update `QuestionMarkIcon` component with styleguide V2.
+- Feature: Add `nano` and replace `micro` props on `ButtonIcon` component.
+- Breaking change: Replace `size: micro` by `nano` on `ButtonIcon` component.
```

Comment:
On peut détailler les classes impactées. Ex:
- Replace `size="micro"` modifier by `size="nano"` in `ButtonIcon` component
- Replace `.k-ButtonIcon--checked--micro` by `.k-ButtonIcon--checked--nano`
- Replace  `.k-ButtonIcon--empty--micro` by `.k-ButtonIcon--empty--nano`
etc.